### ISSUE TITLE
Nested ufuncs in ExpressionTransformer

### ISSUE THREAD

**Issue by bindernet** [jpmml/sklearn2pmml#225]
I would like to nest some ufuncs

A minimal example I have is like this:
```
import sklearn2pmml
from sklearn2pmml.preprocessing import ExpressionTransformer
import numpy

pipe = sklearn2pmml.make_pmml_pipeline(
    ExpressionTransformer("2 if X[0] < 1 else numpy.fmin(1 , numpy.fmin(X[0], X[0]))"),
    ['x1']

)
pipe.fit(np.array([[1]]))

sklearn2pmml.sklearn2pmml(pipe, 'out.pmml')
```

I want to get the minimum of 3 plus numbers. 
I know about the aggregator transformer, but would like to embed it in an if statement, and my downstream users would expect to use pure Python as much as possible for arbitrary logic. 

So my question is:
Is there a workaround or any plan to implement something like this?
Is there a workaround?

EDIT: Just realized this should probably be under jpmml-python ...

**Comment by vruusmann** [comment]
There is currently no limit to UFunc nesting depth. Something like `numpy.exp(numpy.exp(numpy.exp(X[0])))` should be fine.

However, the number of UFunc arguments is limited to one.

> EDIT: Just realized this should probably be under jpmml-python ...

Closing as an exact duplicate of https://github.com/jpmml/jpmml-python/issues/1

I'm building a reusable Python-to-PMML conversion library there, because I'm aiming at some other Python ML frameworks besides Scikit-Learn.

Hope to work on that particular issue very soon.

**Comment by vruusmann** [comment]
Should be all fixed in the newly released SkLearn2PMML version 0.58.0.

However, please note that transformer-only pipelines are currently not supported (will result in a conversion error). There must be a final estimator step available.
